src/dash-adapter.js

```diff
@@ -620,12 +628,26 @@ export default class DashAdapter extends BaseMediaSourceAdapter {
           });
           break;
         case shaka.net.NetworkingEngine.RequestType.MANIFEST:
+          this._parseManifest(response.data);
           this._trigger(EventType.MANIFEST_LOADED, {miliSeconds: response.timeMs});
-          break;
```

Comment:
I would keep it to be on the safe side.
if tomorrow someone will add new case it will be a bug

Reply:
probably removed by mistake.